src/components/Info/Form.js

```diff
@@ -1,15 +1,30 @@
-import React, { useState } from 'react'
-import { Button, Form } from './elements'
+import React from 'react'
+import { useContext } from 'react'
+import { useReducer } from 'react'
+import { StatusDispatch } from '.'
+import { Button, Form, AlertBox, AlertCloseBtn } from './elements'
 
-export default ({ onSubmit, city }) => {
-  const [name, setName] = useState('')
-  const [email, setEmail] = useState('')
-  const [gh, setGH] = useState('')
-  const [plusOne, setPlusOne] = useState(false)
-  const [plusOneName, setPlusOneName] = useState('')
-  const [plusOneGH, setPlusOneGH] = useState('')
-  // TODO: use error to change the front end state, right now it just fails silently if there was an issue persisting the data to Airtable.
-  const [error, setError] = useState(false)
+const formReducer = (state, action) => {
+  const { type, payload } = action
+  return { ...state, [type]: payload }
+}
+
+export default ({ city, error }) => {
+
+  const initialState = {
+    name: '',
+    email: '',
+    gh: '',
+    plusOne: '',
```

Comment:
```suggestion
    plusOne: false,
```
The initial value for `plusOne` was `false` and I believe the code still expects it to be a boolean value instead of a string.

Reply:
My bad.....have reverted it to false